test/DebugInfo/ImportClangSubmodule.swift

```diff
@@ -33,10 +33,10 @@ import OtherClangModule.SubModule
 // submodule's top-level-module, even if we didn't ask for it.
 // CHECK-NOT: !DIImportedEntity({{.*}}, entity: ![[SUBMODULE]]
 // CHECK-NOT: !DIImportedEntity({{.*}}, entity: ![[OTHERSUBMODULE]]
-// CHECK: !DIImportedEntity({{.*}}, entity: ![[CLANGMODULE]])
+// CHECK: !DIImportedEntity({{.*}}, entity: ![[CLANGMODULE]]
 // CHECK-NOT: !DIImportedEntity({{.*}}, entity: ![[SUBMODULE]]
 // CHECK-NOT: !DIImportedEntity({{.*}}, entity: ![[OTHERSUBMODULE]]
-// CHECK: !DIImportedEntity({{.*}}, entity: ![[OTHERCLANGMODULE]])
+// CHECK: !DIImportedEntity({{.*}}, entity: ![[OTHERCLANGMODULE]]
```

Comment:
@adrian-prantl, just double-checking that this is okay? (The entities have grown a `file` field, but that doesn't seem like the important part anyway.)